pkg/k8sutil/openshift_schemes.go

```diff
@@ -0,0 +1,87 @@
+package k8sutil
+
+import (
+	openshiftAppsV1 "github.com/openshift/api/apps/v1"
+	openshiftAppsScheme "github.com/openshift/client-go/apps/clientset/versioned/scheme"
+	openshiftAuthorizationScheme "github.com/openshift/client-go/authorization/clientset/versioned/scheme"
+	openshiftBuildScheme "github.com/openshift/client-go/build/clientset/versioned/scheme"
+	openshiftConfigScheme "github.com/openshift/client-go/config/clientset/versioned/scheme"
+	openshiftConsoleScheme "github.com/openshift/client-go/console/clientset/versioned/scheme"
+	openshiftImageScheme "github.com/openshift/client-go/image/clientset/versioned/scheme"
+	openshiftImageRegistryScheme "github.com/openshift/client-go/imageregistry/clientset/versioned/scheme"
+	openshiftNetworkScheme "github.com/openshift/client-go/network/clientset/versioned/scheme"
+	openshiftOAuthScheme "github.com/openshift/client-go/oauth/clientset/versioned/scheme"
+	openshiftOperatorScheme "github.com/openshift/client-go/operator/clientset/versioned/scheme"
+	openshiftProjectScheme "github.com/openshift/client-go/project/clientset/versioned/scheme"
+	openshiftQuotaScheme "github.com/openshift/client-go/quota/clientset/versioned/scheme"
+	openshiftRouteScheme "github.com/openshift/client-go/route/clientset/versioned/scheme"
+	openshiftSecurityScheme "github.com/openshift/client-go/security/clientset/versioned/scheme"
+	openshiftServiceCertSignerScheme "github.com/openshift/client-go/servicecertsigner/clientset/versioned/scheme"
+	openshiftTemplateScheme "github.com/openshift/client-go/template/clientset/versioned/scheme"
+	openshiftUserScheme "github.com/openshift/client-go/user/clientset/versioned/scheme"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	defaultGroupName = ""
+)
+
+var (
+	// This will ensure that we also support old DeploymentConfigs which are not in the "apps.openshift.io/v1" group
+	// but rather within the core group (for more info see: https://issues.redhat.com/browse/ROX-7515)
```

Comment:
I wonder if we should stop linking to Red Hat internal issue trackers now. Not sure, the context is nice but might discourage open source contributions.

Reply:
I think we actually shall go the other way round: open up issues (an account will still be required) to the world by default.